packages/@aws-cdk/aws-appsync/README.md

```diff
@@ -158,4 +158,16 @@ api.grantMutation(role, 'updateExample');
 
 // For custom types and granular design
 api.grant(role, appsync.IamResource.ofType('Mutation', 'updateExample'), 'appsync:GraphQL');
-```
\ No newline at end of file
+```
+
+## AppSync with X-Ray Tracing
+
+```ts
+const api = new appsync.GraphQLApi(stack, 'Api', {
+  name: 'demo',
+  schemaDefinitionFile: join(__dirname, 'schema.graphql'),
+  xrayEnabled: true,
+});
+```
+
+See the  [the AWS documentation](https://docs.aws.amazon.com/appsync/latest/devguide/x-ray-tracing.html) to learn more about AWS AppSync's X-Ray support.
```

Comment:
I would rather us either put a description of what x-ray support is or not include it at all in the README.

Since it is a feature, I think in order to avoid the linter error, I would be okay with just adding the prop to the [`example`](https://github.com/aws/aws-cdk/blob/master/packages/%40aws-cdk/aws-appsync/README.md#example) section.

Like as follows:
```ts
const api = new appsync.GraphQLApi(stack, 'Api', {
  name: 'demo',
  schemaDefinition: appsync.SchemaDefinition.FILE,
  schemaDefinitionFile: join(__dirname, 'schema.graphql'),
  authorizationConfig: {
    defaultAuthorization: {
      authorizationType: appsync.AuthorizationType.IAM
    },
  },
  xrayEnabled: true,
});
```

@MrArnoldPalmer wdyt?

Reply:
I'll fix it as commented.